openassessment/xblock/static/sass/oa/utilities/_developer.scss

```diff
@@ -135,27 +136,121 @@ $link-hover: $edx-blue-l1 !default; // from our Pattern Library http://ux.edx.or
   }
 
   .student__answer__display__content {
-    border: 1px solid rgba($color-decorative-staff, 0.25);
-    @include padding(($baseline-v/2), ($baseline-h/2), ($baseline-v/2), ($baseline-h/2));
+    @include padding(($baseline-v/2), 0, ($baseline-v/2), 0);
     margin-bottom: ($baseline-v/2);
   }
 
-  .openassessment__student-info_list {
+  .staff-info__student__report {
     list-style-type: none;
+
+    h2.title {
+      margin-top: ($baseline-v/2);
+      border-top: 1px solid $heading-staff-color;
+    }
   }
 
   .value {
     width: $max-width/2;
   }
 
-  /**
-   * The follow styles are bound for the "shame" file. This is done to override
-   * LMS specific styles on HTML elements.
-   */
+  .ui-staff__action__section {
+    padding: $baseline-v ($baseline-h/2);
+    background-color: white;
+    margin-bottom: ($baseline-v/2);
+
+    .ui-staff__title {
+      text-transform: none;
+      font-size: 18px;
```

Comment:
@andy-armstrong please also take a look at this feedback (I suspect much of this CSS will change or be deleted since Frances would like the last 2 sections to look like the rest when collapsed (black).